core/src/cluster_slots_service.rs

```diff
@@ -79,6 +79,14 @@ impl ClusterSlotsService {
         cluster_slots_update_receiver: ClusterSlotsUpdateReceiver,
         exit: Arc<AtomicBool>,
     ) {
+        let node_id = cluster_info.id();
+        let my_stake = bank_forks
```

Comment:
I believe we need to derive each of these in the loop since they can change

Reply:
addressed in [df0ffb9](https://github.com/anza-xyz/agave/pull/5141/commits/df0ffb94e900d948431c764fe1f6cbc7ff53f63f)